pkg/crossnodepreemption/candidate.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crossnodepreemption
+
+import (
+	v1 "k8s.io/api/core/v1"
+	extenderv1 "k8s.io/kube-scheduler/extender/v1"
+)
+
+type candidate struct {
+	victims []*v1.Pod
+	name    string
+}
+
+// Victims returns s.victims.
+func (s *candidate) Victims() *extenderv1.Victims {
+	return &extenderv1.Victims{
+		Pods:             s.victims,
+		NumPDBViolations: 0,
```

Comment:
The intention is to make `candidate` irrelevant with the `extenderv1` pkg, so that in the preemption logic, we don't need to worry about `extenderv1` pkg. And returning `extenderv1.Victims` here does nothing magic but ensuring it adheres to `Candidate` interface.